src/variant.cpp

```diff
@@ -701,7 +701,9 @@ void from_variant( const variant& var, uint128_t& vo, uint32_t max_depth )
 #endif
 }
 
-#ifdef __APPLE__
+#if defined(__APPLE__)
+#elif defined(__OpenBSD__)
+   void to_variant( size_t s, variant& v, uint32_t max_depth ) { v = variant( int64_t(s) ); }
```

Comment:
Not sure whether this is really needed. I suspect it's unused since macOS build doesn't rely on it.

Reply:
OK, found the code for macOS in `variant.hpp`.

Reply:
Using `int64_t` but not `uint64_t` here is probably wrong.

Reply:
Fix: https://github.com/bitshares/bitshares-fc/pull/194.